src/CarrierLogo/CarrierLogo.js.flow

```diff
@@ -0,0 +1,14 @@
+// @flow
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+
+import { SIZE_OPTIONS, CARRIER_TYPE_OPTIONS } from "./consts";
+
+type Carrier = { code: string, name: string, type?: $Values<typeof CARRIER_TYPE_OPTIONS> };
```

Comment:
$Values will not work, can you please specify it ? like "airline" | "bus" | "train"